packages/styles/stories/Dev/facets/combined.example.html

```diff
@@ -5,7 +5,7 @@
             </span>
         </div>
         <div class="fd-facet fd-facet--form fd-margin-end--md fd-margin-bottom--sm" role="group" aria-labelledby="formFacetTitle1">
-            <h1 class="fd-title fd-title--h5 fd-margin-bottom--sm" id="formFacetTitle1">Technical Data</h1>
+            <span role="heading" aria-level="3" class="fd-title fd-margin-bottom--sm" id="formFacetTitle1">Technical Data</span>
```

Comment:
you can use `div` and then you won't need `display: block;` in the CSS. Confirmed with Nikolay K.